lib/compat/wordpress-6.7/rest-api.php

```diff
@@ -156,3 +156,69 @@ function gutenberg_register_post_type_args_for_wp_global_styles( $args, $post_ty
 }
 
```

Comment:
It's too late for WP 6.7 now. This would need to be a target for 6.8. But I'd wait to refresh until it becomes more clear this is something to include.